suite/tests/linux/syscall_pwait.cpp

```diff
@@ -278,31 +280,31 @@ main(int argc, char *argv[])
          * of the call and restore it post syscall. The mask check is making sure
          * that save/restore is done properly.
          */
-        asm volatile("mov %7, %%rbx\n\t"
-                     "movq %2, %%rax\n\t"
-                     "movq %3, %%rdi\n\t"
-                     "movq %4, %%rsi\n\t"
-                     "movq %5, %%rdx\n\t"
-                     "movq %6, %%r10\n\t"
-                     "movq %7, %%r8\n\t"
-                     "movq %8, %%r9\n\t"
-                     "syscall\n\t"
-                     "mov $0, %0\n\t"
-                     "cmp $-4095, %%rax\n\t"
-                     "jl no_syscall_error%=\n\t"
-                     "mov $1, %0\n\t"
-                     "no_syscall_error%=:\n\t"
-                     "mov $0, %1\n\t"
-                     "cmp %%rbx, %%r8\n\t"
-                     "je no_mask_error%=\n\t"
-                     "mov $1, %1\n\t"
-                     "no_mask_error%=:\n"
-                     /* early-clobber outputs */
-                     : "=&r"(syscall_error), "=&r"(mask_error)
-                     : "r"((int64_t)SYS_epoll_pwait), "rm"((int64_t)epoll_fd),
-                       "rm"(&events), "r"(24LL), "r"(-1LL), "rm"(&test_set),
-                       "r"((int64_t)(_NSIG / 8))
-                     : "rax", "rdi", "rsi", "rdx", "r10", "r8", "r9", "rbx");
+        asm volatile(
+            "mov %7, %%rbx\n\t"
+            "movq %2, %%rax\n\t"
+            "movq %3, %%rdi\n\t"
+            "movq %4, %%rsi\n\t"
+            "movq %5, %%rdx\n\t"
+            "movq %6, %%r10\n\t"
+            "movq %7, %%r8\n\t"
+            "movq %8, %%r9\n\t"
+            "syscall\n\t"
+            "mov $0, %0\n\t"
+            "cmp $-4095, %%rax\n\t"
+            "jl no_syscall_error%=\n\t"
+            "mov $1, %0\n\t"
+            "no_syscall_error%=:\n\t"
+            "mov $0, %1\n\t"
+            "cmp %%rbx, %%r8\n\t"
+            "je no_mask_error%=\n\t"
+            "mov $1, %1\n\t"
+            "no_mask_error%=:\n"
+            /* early-clobber outputs */
+            : "=&r"(syscall_error), "=&r"(mask_error)
```

Comment:
Please rewrite this using named parameters.  This will make it easier to read so we don't have to manually figure out which number is which parameter.  See suite/tests/linux/rseq.c for examples.

Reply:
Done for all asm volatiles.